http://bugs.gentoo.org/361057 for GSmartPointer.h change

Then include stddef.h in the installed headers to fix building of reverse
dependencies, such as app-text/zathura-djvu

These are all in upstream GIT

--- a/libdjvu/ddjvuapi.h
+++ b/libdjvu/ddjvuapi.h
@@ -64,6 +64,7 @@
 }
 #endif
 
+#include <stddef.h>
 #include <stdlib.h>
 #include <stdio.h>
 
--- a/libdjvu/GSmartPointer.h
+++ b/libdjvu/GSmartPointer.h
@@ -62,6 +62,8 @@
 # pragma interface
 #endif
 
+#include <cstddef>
+
 /** @name GSmartPointer.h
 
     Files #"GSmartPointer.h"# and #"GSmartPointer.cpp"# define a smart-pointer
--- a/libdjvu/miniexp.h
+++ b/libdjvu/miniexp.h
@@ -38,7 +38,8 @@
 #ifndef MINILISPAPI
 # define MINILISPAPI /**/
 #endif
-  
+
+#include <stddef.h>
 
 /* -------------------------------------------------- */
 /* LISP EXPRESSIONS                                   */
